code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -0,0 +1,328 @@
+
+/obj/item/weapon/nullrod/hammmer
+	icon_state = "hammeron"
+	item_state = "hammeron"
+	name = "relic war hammer"
+	desc = "This war hammer cost the chaplain fourty thousand space dollars."
+	slot_flags = SLOT_BELT
+	w_class = 5
+	attack_verb = list("smashed", "bashed", "hammered", "crunched")
+
+/obj/item/weapon/nullrod/chainsaw
+	name = "chainsaw hand"
+	desc = "Good? Bad? You're the guy with the chainsaw hand."
+	icon_state = "chainsaw1"
+	item_state = "mounted_chainsaw"
+	w_class = 5
+	flags = NODROP | ABSTRACT
+	sharp = 1
+	edge = 1
+	attack_verb = list("sawed", "torn", "cut", "chopped", "diced")
+	hitsound = 'sound/weapons/chainsaw.ogg'
+
+/obj/item/weapon/nullrod/clown
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "clownrender"
+	item_state = "gold_horn"
+	name = "clown dagger"
+	desc = "Used for absolutely hilarious sacrifices."
+	hitsound = 'sound/items/bikehorn.ogg'
+	sharp = 1
+	edge = 1
+	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+
+/obj/item/weapon/nullrod/whip
+	name = "holy whip"
+	desc = "What a terrible night to be on Space Station 13."
+	icon_state = "chain"
+	item_state = "chain"
+	slot_flags = SLOT_BELT
+	attack_verb = list("whipped", "lashed")
+	hitsound = 'sound/weapons/slash.ogg'
+
+/obj/item/weapon/nullrod/whip/afterattack(atom/movable/AM, mob/user, proximity)
+	if(!proximity)
+		return
+	if(istype(AM, /mob/living/carbon/human))
+		var/mob/living/carbon/human/H = AM
+		if(is_shadow(H))
+			var/phrase = pick("Die monster! You don't belong in this world!!!", "You steal men's souls and make them your slaves!!!", "Your words are as empty as your soul!!!", "Mankind ill needs a savior such as you!!!")
+			user.say("[phrase]")
+			H.adjustBruteLoss(8) //Bonus damage
+
+/obj/item/weapon/nullrod/fedora
+	name = "athiest's fedora"
+	desc = "The brim of the hat is as sharp as your wit. Throwing it at someone would hurt almost as much as disproving the existence of God."
+	icon_state = "fedora"
+	item_state = "fedora"
+	slot_flags = SLOT_HEAD
+	icon = 'icons/obj/clothing/hats.dmi'
+	force = 0
+	throw_speed = 4
+	throw_range = 7
+	throwforce = 20
+
+/obj/item/weapon/nullrod/armblade
+	name = "dark blessing"
+	desc = "Particularly twisted dieties grant gifts of dubious value."
+	icon_state = "arm_blade"
+	item_state = "arm_blade"
+	flags = ABSTRACT | NODROP
+	w_class = 5
+	sharp = 1
+	edge = 1
+
+/obj/item/weapon/nullrod/carp
+	name = "carp-sie plushie"
+	desc = "An adorable stuffed toy that resembles the god of all carp. The teeth look pretty sharp. Activate it to recieve the blessing of Carp-Sie."
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "carpplushie"
+	item_state = "carp_plushie"
+	force = 15
+	attack_verb = list("bitten", "eaten", "fin slapped")
+	hitsound = 'sound/weapons/bite.ogg'
+	var/used_blessing = FALSE
+
+/obj/item/weapon/nullrod/carp/attack_self(mob/living/user)
+	if(used_blessing)
+		return
+	if(user.mind && (user.mind.assigned_role != "Chaplain"))
```

Comment:
I added this check to prevent assistants breaking in and making a god hand, rendering the weapon unable to be recovered